lib/exqlite/sqlite3.ex

```diff
@@ -129,26 +140,7 @@ defmodule Exqlite.Sqlite3 do
     #
     # For now this just works
     chunk_size = Application.get_env(:exqlite, :default_chunk_size, 50)
-    fetch_all(conn, statement, chunk_size, [])
-  end
-
-  defp fetch_all(conn, statement, chunk_size, accum) do
-    case multi_step(conn, statement, chunk_size) do
-      {:done, rows} ->
-        case accum do
-          [] -> {:ok, rows}
-          accum -> {:ok, Enum.reverse(rows ++ accum)}
```

Comment:
rows should've been reversed here before concating, I think

Reply:
You are right. They should have been reversed, before appending.